compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/ConversionUtils.kt

```diff
@@ -86,16 +87,17 @@ class ConversionTypeContext internal constructor(
 fun FirClassifierSymbol<*>.toSymbol(
     session: FirSession,
     classifierStorage: Fir2IrClassifierStorage,
-    typeContext: ConversionTypeContext = ConversionTypeContext.DEFAULT
+    typeContext: ConversionTypeContext = ConversionTypeContext.DEFAULT,
+    handleAnnotations: ((List<FirAnnotationCall>) -> Unit)? = null
```

Comment:
In general PR looks Ok but I don't like this callback, this place looks hacky a bit. I will try to rewrite this place differently.

Reply:
Left as is